install-k8s-perf.yml

```diff
@@ -1,7 +1,7 @@
 - hosts: all
   tasks:
     - set_fact:
-        prepull_images: "{{ prepull_images + ['registry.k8s.io/pause:3.1'] }}"
+        prepull_images: "{{ prepull_images + ['registry.k8s.io/pause:3.9'] }}"
```

Comment:
The latest code on test-infra/perf-tests points to pause:3.9, updating the same here to pre-pull the correct image.